sandbox/registration-processor-mz.properties

```diff
@@ -677,6 +702,16 @@ mosip.regproc.printing.server.port=8099
 mosip.regproc.printing.server.servlet.path=/registrationprocessor/v1/print-stage
 mosip.regproc.printing.eventbus.port=5722
 
+#workflow-manager-service
+mosip.regproc.workflow.action.job.eventbus.kafka.commit.type=single
+mosip.regproc.workflow.action.job.eventbus.kafka.max.poll.records=100
+mosip.regproc.workflow.action.job.eventbus.kafka.poll.frequency=100
```

Comment:
Can we avoid kafka and server port properties for the job since it does not connect kafka ? and does not expose an api ?

Reply:
fixed